.github/actions/mockTest/action.yml

```diff
@@ -1,6 +1,14 @@
-name: "Mock Test"
+name: "Mock Tests"
 description: "Run mock tests"
-
 runs:
-  using: "node12"
-  main: "./main.js"
+  using: "composite"
+  steps:
+    - run: ./node_modules/.bin/next dev -p 8080 &
+      shell: bash
+      env:
+        NEXT_PUBLIC_API_URL: https://api.replay.io/v1/graphql
+        NEXT_PUBLIC_DISPATCH_URL: wss://dispatch.replay.io
```

Comment:
Let's move these to a `.env.sample` and then just `cp` that in this job. This should look as close as possible to a new dev setup following the README.